diary/index.md

```diff
@@ -1,3 +1,6 @@
 # My diary
 
 My programming oddysey starts here :)
+
+The real index of the pages living here can be found
+[here](contents).
```

Comment:
I'm redirecting to `diary/contents`, but I _think_ the "real index" should be here. But that may belong to a future PR.